python/taichi/shaders/SceneLines_vk.vert

```diff
@@ -0,0 +1,43 @@
+#version 450
+
+layout(location = 0) in vec3 in_position;
+layout(location = 1) in vec3 in_normal;
+layout(location = 2) in vec2 in_texcoord;
+layout(location = 3) in vec4 in_color;
+
+layout(location = 0) out vec2 frag_texcoord;
+
+struct SceneUBO {
+  vec3 camera_pos;
+  mat4 view;
+  mat4 projection;
+  vec3 ambient_light;
+  int point_light_count;
+};
+
+struct PointLight {
+  vec3 pos;
+  vec3 color;
+};
+
+layout(binding = 0) uniform UBO {
+  SceneUBO scene;
+  vec3 color;
+  int use_per_vertex_color;
+}
+ubo;
```

Comment:
There wasn't a `scene` member in the vertex program, wouldn't this cause any problem?